regparser/commands/outline_depths.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from regparser.tree.depth.derive import derive_depths
+
+import click
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument('markers', type=click.STRING, required=True)
```

Comment:
No biggie, but if you set `[nargs=-1](http://click.pocoo.org/5/arguments/#variadic-arguments)`, users won't need to quote their parameter:
```bash
eregs markers 1 2 3 4 5
# vs
eregs markers "1 2 3 4 5"
```